k8s/deployment.yaml

```diff
@@ -30,6 +30,21 @@ roleRef:
 
 ---
 
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: agent-sandbox-controller-extensions
+subjects:
+- kind: ServiceAccount
+  name: agent-sandbox-controller
+  namespace: agent-sandbox-system
+roleRef:
+  kind: ClusterRole
+  name: agent-sandbox-controller-extensions
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
This needs to be separated? e.g. use kustomize

Reply:
will add a separate PR for kustomize. 
For now separated this cluster role into a spearate file.